sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/state/metadata/StateMetadataSource.scala

```diff
@@ -47,7 +47,8 @@ case class StateMetadataTableEntry(
     minBatchId: Long,
     maxBatchId: Long,
     operatorPropertiesJson: String,
-    numColsPrefixKey: Int) {
+    numColsPrefixKey: Int,
```

Comment:
nit: Do we want to move the stateSchemaFilePath before numColsPrefixKey, since the latter is a metadata column?

Reply:
I guess its fine to just add at the end since its a new field ?

Reply:
Sure, sounds good